airflow/models/baseoperator.py

```diff
@@ -1874,3 +1877,21 @@ def get_link(self, operator: BaseOperator, *, ti_key: TaskInstanceKey) -> str:
         :param ti_key: TaskInstance ID to return link for.
         :return: link to external system
         """
+
+
+class BaseDeferrableOperator(BaseOperator):
+    """Abstract base class for deferrable operators.
```

Comment:
This does not seem to be a true Abstract class as it's not extending `abc.ABC` and users might be able to create instances of this class. Can we recheck the wording here please based on whether we want to allow the creation of instances for this class or not? Or extend it from `abc.ABC`? WDYT?

Reply:
Yep, I think we should reword here. Thanks!